pkg/config/validation/validation_test.go

```diff
@@ -313,15 +342,18 @@ func TestValidateProtocolDetectionTimeout(t *testing.T) {
 	}
 }
 
+// refactor
```

Comment:
If this is a followup generally we put `// TODO: `

Reply:
Removed.